gobgp/cmd/global.go

```diff
@@ -809,8 +811,21 @@ func ParsePath(rf bgp.RouteFamily, args []string) (*table.Path, error) {
 		if err != nil {
 			return nil, err
 		}
-		p := bgp.NewPathAttributeExtendedCommunities(extcomms)
+		normalextcomms := make([]bgp.ExtendedCommunityInterface, 0)
+		ipv6extcomms := make([]bgp.ExtendedCommunityInterface, 0)
+		for _, com := range extcomms {
+			bstring, _ := com.Serialize()
+			switch len(bstring) {
+			case 20:
```

Comment:
How about using `com.GetTypes()` to check if we need to put this community as an IPv6 extended community path attribute?